web/models/user.ex

```diff
@@ -0,0 +1,29 @@
+defmodule MovieWagerApi.User do
+  use MovieWagerApi.Web, :model
+
+  schema "users" do
+    field :username, :string
+    field :profile_image_url, :string
+    timestamps()
+  end
+
+  @doc """
+  Builds a changeset based on the `struct` and `params`.
+  """
+  def changeset(struct, params \\ %{}) do
+    struct
+    |> cast(params, [:username, :profile_image_url])
+    |> validate_required([:username])
+  end
+
+  def by_id_or_username(identifier) when is_integer(identifier) do
+    from u in MovieWagerApi.User, where: u.id == ^identifier
+  end
+
+  def by_id_or_username(identifier) when is_binary(identifier) do
```

Comment:
The idea here is that we will want to be able to easily query a user by id or username.